spec/reek/configuration/configuration_file_finder_spec.rb

```diff
@@ -4,6 +4,8 @@
 require 'tmpdir'
 require_relative '../../spec_helper'
 
+include Reek::Configuration
+
```

Comment:
I guess we need this so we can run the spec file individually – can we instead `describe Reek::Configuration::ConfigurationFileFinder` and use `described_class` where `ConfigurationFileFinder` is used in the specs?

(I’m ok with the `include` if that’s your preference, but in general I prefer not to mess with namespacing more than necessary.)